pkg/scheduler/apis/config/types.go

```diff
@@ -209,3 +209,33 @@ type PluginConfig struct {
 	// Args defines the arguments passed to the plugins at the time of initialization. Args can have arbitrary structure.
 	Args runtime.Unknown
 }
+
+func appendPluginSet(dst *PluginSet, src *PluginSet) *PluginSet {
+	if dst == nil {
+		dst = &PluginSet{}
+	}
+	if src != nil {
+		dst.Enabled = append(dst.Enabled, src.Enabled...)
+		dst.Disabled = append(dst.Disabled, src.Disabled...)
+	}
+	return dst
+}
+
+// Append appends src Plugins to current Plugins. If a PluginSet is nil, it will
+// be created.
+func (p *Plugins) Append(src *Plugins) {
```

Comment:
Does this need to be exposed to `staging` for user consuming? Or just for internal usage.

Reply:
Just internal use, kube-proxy does that as well.

Reply:
OK, just want to confirm on this.

Reply:
added a comment to clarify that.